Strange error with eslint

I just installed **polished**, and ❤️  it (just like **styled-components**)!

There is just one little bug (?) that bothers me..
I use **eslint** for linting, and it keeps saying me that I have 2 errors for each `import polished from 'polished'`:
```
/polished-test/src/Component.js
  2:32  error  Unable to resolve path to module 'polished'  import/no-unresolved
  2:32  error  Missing file extension for "polished"        import/extensions
```

My guess is that in the npm name there is an invisible character..

Here's how to reproduce it:
```bash
git clone https://github.com/EnoahNetzach/polished-test
cd polished-test
yarn
yarn eslint
```

--- a/package.json
+++ b/package.json
@@ -2,8 +2,8 @@
   "name": "polished",
   "description": "A lightweight toolset for writing styles in Javascript.",
   "main": "lib/index.js",
-  "jsnext:main": "dist/styled-components.es.js",
-  "module": "dist/styled-components.es.js",
+  "jsnext:main": "dist/polished.es.js",
+  "module": "dist/polished.es.js",
   "scripts": {
     "build": "npm run build:lib && npm run build:dist && npm run build:flow && npm run build:docs",
     "prebuild:lib": "shx rm -rf lib/*",

